### PR TITLE
Update readmes and pins for helm3 support

### LINKED ISSUE #462: azure-ts-aks-mean fails due to older version of helm
https://github.com/pulumi/examples/tree/master/azure-ts-aks-mean sample is currently failing as the  new version of `helm` (3.0) doesn't seem to support a few of the commands that the sample depends on.

For example, the `helm init --client-only` option doesn't exist, `helm --fetch` has been replaced by `helm --pull` and couple more downstream issues.

Perhaps you can add a note to the README.md for now that the sample only works with helm 2.0 while you track this backlog item to address the compat issues?

### PR DESCRIPTION
**Pull request by lblackstone**
Fixes #462

### CODE REVIEW

File: azure-ts-aks-keda/README.md [hunk 1]
```diff
@@ -15,8 +15,12 @@ We will be deploying to Azure, so you will need an Azure account. If you don't h
 This example deploys a Helm Chart from Kedacore Helm chart repository, so you
 will need to [install the Helm CLI](https://docs.helm.sh/using_helm/#installing-helm) and configure it:
 
+If you are using Helm v2:
 ```bash
 $ helm init --client-only
+```
+
+```
```
Comment (lukehoban) [inline azure-ts-aks-keda/README.md]: This one is stylistically different than the others - and perhaps doesn’t make clear this step is needed for Helm 3 as well?

File: kubernetes-ts-helm-wordpress/README.md [hunk 2]
```diff
@@ -17,17 +17,11 @@ Helm Chart and submit the expanded YAML to the cluster.
...
 
 ```sh
-brew install kubernetes-helm
-```
-
-Once `helm` is installed, initialize it with:
-
-```sh
-helm init --client-only
+brew install helm
```
Comment (lukehoban) [inline kubernetes-ts-helm-wordpress/README.md]: Is this right?  I though the formula was still `kubernetes-helm`?

File: azure-ts-aks-keda/README.md (hunk 1 above)
Comment (lblackstone) [reply azure-ts-aks-keda/README.md]: Updated

File: kubernetes-ts-helm-wordpress/README.md (hunk 2 above)
Comment (lblackstone) [reply kubernetes-ts-helm-wordpress/README.md]: Yes, they renamed the formula

**Comment by lblackstone** [comment]
Test failures unrelated to these changes. Merging.

**Pull request merged**